openstack/02a-MAClow/IEEE802154E.h

```diff
@@ -36,20 +36,25 @@ static const uint8_t ebIEsBytestream[] = {
 #define EB_SLOTFRAME_NUMLINK_OFFSET 22
 
 #define EB_IE_LEN                   28
-
 #define NUM_CHANNELS                16 // number of channels to channel hop on
 #define TXRETRIES                   15 // number of MAC retries before declaring failed
 #define TX_POWER                    31 // 1=-25dBm, 31=0dBm (max value)
 #define RESYNCHRONIZATIONGUARD       5 // in 32kHz ticks. min distance to the end of the slot to successfully synchronize
 #define US_PER_TICK                 30 // number of us per 32kHz clock tick
-#define EB_PORTION                  10 // set EB on minimal cell for 1/EB_PORTION portion
 #define MAXKAPERIOD               1000 // in slots: 1500@20ms per slot -> ~30 seconds. Max value used by adaptive synchronization.
 #define DESYNCTIMEOUT             1750 // in slots: 1750@20ms per slot -> ~35 seconds. A larger DESYNCTIMEOUT is needed if using a larger KATIMEOUT.
 #define LIMITLARGETIMECORRECTION     5 // threshold number of ticks to declare a timeCorrection "large"
 #define LENGTH_IEEE154_MAX         128 // max length of a valid radio packet
 #define DUTY_CYCLE_WINDOW_LIMIT    (0xFFFFFFFF>>1) // limit of the dutycycle window
 #define SERIALINHIBITGUARD          32 // 32@32kHz ~ 1ms
 
+// set EB on minimal cell for 1/EB_PORTION portion

```

Comment:
You can just change EB_PORTION value when necessary at local.
No need to commit this changes.

Reply:
ok, removed it but added a comment before this configuration to explain that it is connected to the change in slot duration. Otherwise synchronizations become nearly impossible for longer slot duration